### PR TITLE
Created group theme to use for project graphs.

### PR DESCRIPTION
**Pull request by FarhanMajumder**

### CODE REVIEW

**Review by RuiZhou96 — APPROVED** [review]

**Review by leilakrichel — APPROVED** [review]

**Review by jeff-a-nguyen — APPROVED** [review]

**Review by leilakrichel — APPROVED** [review]

**Review by jeff-a-nguyen — APPROVED** [review]

File: R/graph_theme.R [hunk 1]
```diff
@@ -0,0 +1,11 @@
+jrc_theme <- function() {
+    theme(
+        plot.title = element_text(hjust = 0.5),
+        panel.border = element_rect(fill = NA),
+        axis.line = element_line(),
+        text = element_text(size = 11, family = "Georgia"),
```
Comment (QuLogic) [inline R/graph_theme.R:6]: Did you try this on all your computers? I'm not sure this font exists everywhere.

Comment (jeff-a-nguyen) [reply R/graph_theme.R:6]: We did it on Farhan's Mac and it worked. We did not try any other computer.

Comment (lwjohnst86) [reply R/graph_theme.R:6]: That should be fine for now. If there are problems later on, we can help figure it out. Enjoy your fall break!

**Comment by lwjohnst86** [comment]
@FarhanMajumder if you want, you can merge this in.

**Pull request merged**